packages/vm/lib/index.ts

```diff
@@ -225,6 +232,8 @@ export default class VM extends AsyncEventEmitter {
       return
     }
 
+    await this.blockchain.initPromise
```

Comment:
Can't you just use your static constructor here?

Reply:
Update: ah, got it, `Blockchain` is created in the constructor.

Reply:
I will remove this since all public `blockchain` methods await this one internally; just bloats the logic here.

Reply:
Never mind; if you don't await this Promise then after `init` VM might be initialized, but `Blockchain` might not be initialized.